bug: `ghokin`: [ERROR] Formatter 'ghokin' error: you must provide a filename as argument

### Neovim version (nvim -v)

NVIM 0.11.4

### Operating system/version

MacOS 15.6.1

### Read debugging tips

- [x] I have read through the [debugging tips](https://github.com/stevearc/conform.nvim/blob/master/doc/debugging.md).

### Add the debug logs

- [x] I have set `log_level = vim.log.levels.DEBUG` and pasted the log contents below.

### Log file

```
2025-09-08 10:25:07[DEBUG] Running formatters on /Users/drew/projects/conform-ghokin-repro/test.feature: { "ghokin" }
2025-09-08 10:25:07[INFO] Run ghokin on /Users/drew/projects/conform-ghokin-repro/test.feature
2025-09-08 10:25:07[DEBUG] Run command: { "/Users/drew/.local/share/mise/installs/go/1.24.4/bin/ghokin", "fmt", "stdout" }
2025-09-08 10:25:07[DEBUG] Run default CWD: /Users/drew/projects/conform-ghokin-repro
2025-09-08 10:25:07[INFO] ghokin exited with code 1
2025-09-08 10:25:07[DEBUG] ghokin stdout: { "" }
2025-09-08 10:25:07[DEBUG] ghokin stderr: { "you must provide a filename as argument", "" }
2025-09-08 10:25:07[ERROR] Formatter 'ghokin' error: you must provide a filename as argument
```

### Describe the bug

Thanks so much for `conform.nvim` - I use this plugin everyday and it's been awesome to use. Really appreciate how easy it is to configure and all of the documentation for it.

I recently moved from using `reformat-gherkin` to `ghokin` to format `.feature` files and noticed that after configuring `ghokin` to format `cucumber` files in my `conform` config, I was getting an error message saying `Formatter 'ghokin' error: you must provide a filename as argument`.

### What is the severity of this bug?

blocking (cannot use plugin)

### Steps To Reproduce

1. Install `ghokin` using `go install github.com/antham/ghokin@latest`
2. Create a `repro.lua` file (content below)
> NOTE: Creates a custom `:Format` command to run formatters on a buffer, and sets `ghokin` as the formatter for `cucumber` filetypes
4. Create a `test.feature` file (content below)
5. Open the `test.feature` file in `neovim` with: `nvim -u repro.lua test.feature`
6. Run the `:Format` command
7. See that you get an error saying that a filename argument must be supplied

### Expected Behavior

I would expect that when I run `:Format` on a `cucumber` (i.e., a `.feature`) file, after configuring `ghokin` using `conform.lua`, that this plugin would format the file without any errors.

### Minimal example file

`test.feature` (notice the extra indentation on the first scenario line):

```gherkin
Feature: Test API Endpoint

  Scenario: GET /test
        Given I am a user
    When I send a GET request to /test
    Then the response has an HTTP status of 200
```

### Minimal init.lua

```Lua
-- DO NOT change the paths and don't remove the colorscheme

local root = vim.fn.fnamemodify("./.repro", ":p")

-- set stdpaths to use .repro
for _, name in ipairs({ "config", "data", "state", "cache" }) do
	vim.env[("XDG_%s_HOME"):format(name:upper())] = root .. "/" .. name
end

-- bootstrap lazy
local lazypath = root .. "/plugins/lazy.nvim"
if not vim.loop.fs_stat(lazypath) then
	vim.fn.system({
		"git",
		"clone",
		"--filter=blob:none",
		"--single-branch",
		"https://github.com/folke/lazy.nvim.git",
		lazypath,
	})
end
vim.opt.runtimepath:prepend(lazypath)

-- install plugins
local plugins = {
	"folke/tokyonight.nvim",
	{
		"stevearc/conform.nvim",
		init = function()
			vim.api.nvim_create_user_command("Format", function(args)
				local range = nil
				if args.count ~= -1 then
					local end_line = vim.api.nvim_buf_get_lines(0, args.line2 - 1, args.line2, true)[1]
					range = {
						start = { args.line1, 0 },
						["end"] = { args.line2, end_line:len() },
					}
				end
				require("conform").format({ async = true, lsp_fallback = true, range = range })
			end, { range = true })
		end,
		config = function()
			require("conform").setup({
				log_level = vim.log.levels.DEBUG,
				-- add your config here
				formatters_by_ft = {
					lua = { "stylua" },
					cucumber = { "ghokin" },
				},
			})
		end,
	},
	-- add any other plugins here
}
require("lazy").setup(plugins, {
	root = root .. "/plugins",
})

vim.cmd.colorscheme("tokyonight")
-- add anything else here
```

### Additional context

_No response_

--- a/lua/conform/formatters/ghokin.lua
+++ b/lua/conform/formatters/ghokin.lua
@@ -6,5 +6,5 @@ return {
   },
 
   command = "ghokin",
-  args = { "fmt", "stdout" },
+  args = { "fmt", "stdout", "$FILENAME" },
 }

